src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -355,11 +356,21 @@ export const createStageGateChangeRequest = async (req: Request, res: Response)
         projectNumber: body.wbsNum.projectNumber,
         workPackageNumber: body.wbsNum.workPackageNumber
       }
+    },
+    include: {
+      workPackage: { include: { expectedActivities: true, deliverables: true } }
     }
   });
   if (wbsElement === null) {
     return res.status(404).json({ message: `wbs number ${body.wbsNum} not found` });
   }
+  if (wbsElement.workPackage?.expectedActivities || wbsElement.workPackage?.deliverables) {
+    const uncheckedEADBs = wbsElement.workPackage?.expectedActivities.map(checkDBChecked);
```

Comment:
i think this code is too dense. need to find a way to break it down more and name it better. also needs some commenting

```
const uncheckedExpectedActivitiy = expectedActivities.indexOf((ea) => ea.dateTimeChecked === null);
if uncheckedExpectedActivitiy !== -1 => error
```

then the same for deliverables